.rubocop.yml

```diff
@@ -4,3 +4,13 @@ LineLength:
    Max: 99
 Style/FrozenStringLiteralComment:
    Enabled: false
+Style/SymbolArray:
+  Enabled: false
+Rails/Blank:
+  Enabled: false
+Layout/TrailingBlankLines:
+  Enabled: false
+Style/SafeNavigation:
+  Enabled: false
+Style/RescueStandardError:  
+  Enabled: false
```

Comment:
This one too, here you have the description of the cop: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/RescueStandardError

The thing is that doing a `rescue StandardError` is not neccesary because that is the default error.
Then if you have different Errors, you can do that.

This is like the try - catch of other languages.